frontend/public/components/about-modal.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-unused-vars, no-undef */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { AboutModal as PfAboutModal, TextContent, TextList, TextListItem } from '@patternfly/react-core';
+
+import { clusterIDStateToProps } from '../ui/ui-reducers';
+import { getBrandingDetails } from './masthead';
+import { k8sVersion } from '../module/status';
+
+const AboutModal_: React.FC<AboutModalProps> = ({isOpen, closeAboutModal, clusterID}) => {
+  const [kubernetesVersion, setKubernetesVersion] = React.useState('');
+  React.useEffect(() => {
```

Comment:
Effects run every time the component renders, so we should pass the [2nd argument to `useState()` to mimic the behavior of `componentDidMount()`](https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect).